mitodl/mit-open
# search page styling

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4468

### Description (What does it do?)
This pr updates the styling of the search page. The designs are here:
https://www.figma.com/proto/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?page-id=107%3A2479&node-id=3375-49853&viewport=-15774%2C-1090%2C0.55&t=cX0nylncBbGUadPG-1&scaling=min-zoom&starting-point-node-id=2197%3A31345

Out of scope:
- resource cards
- The "learning materials" tab, which will combine videos and podcasts

### Screenshots (if appropriate):
<br>
<br>
<img width="1710" alt="Screenshot 2024-06-10 at 6 35 14 PM" src="https://github.com/mitodl/mit-open/assets/1934992/f1a24e92-c2eb-4a6d-ae43-8600d5308f61">
<br>
<br>
<img width="388" alt="Screenshot 2024-06-10 at 6 35 50 PM" src="https://github.com/mitodl/mit-open/assets/1934992/a7d188ac-481c-4b2c-8c95-368ebea41552">
<br>
<br>
<img width="388" alt="Screenshot 2024-06-10 at 6 36 01 PM" src="https://github.com/mitodl/mit-open/assets/1934992/d9575c82-b713-4b85-add0-f9550e9c9924">


### How can this be tested?
Go to http://localhost:8063/search verify that the page looks good and looks like the designs
